### PR TITLE
Added recruitment experiment tutorial notebook

### LINKED ISSUE #56: Make a Jupyter notebook clearly showing the steps in the recruitment experiment.
This should be done before addressing #47 and the code to follow is: https://github.com/neurodata/progressive-learning/blob/master/experiments/recruitment_exp/experiment.py

### PR DESCRIPTION
**Pull request by emilyachang**
#### Reference issue

Closes #56 

#### Type of change

Feature request

#### What does this implement/fix?

Completed recruitment experiment notebook: https://nbviewer.jupyter.org/github/emilyachang/progressive-learning/blob/recruitment-exp/docs/tutorials/recruitment_exp.ipynb
- Explains/provides background on recruitment experiment
- Updates old recruitment experiment code to use new ProgLearn instances (see recruitment_functions.py)
- Runs recruitment experiment and displays results (reps = 5)

#### Additional information

@jdey4 Let me know if you want me to fix anything in the implementation - I kept the print statements in for now in case you wanted to check it when running over more reps.

### CODE REVIEW

**Review by PSSF23 — COMMENTED** [review]
@jdey4 @levinwil I thought we are using `keras` as a standalone package instead of `tensorflow.keras`? Or is the setting different among benchmarks?

**Comment by levinwil** [comment]
@PSSF23 you are correct: we use Keras, not tensorflow.keras

@emilyachang Instead of doing `import tensorflow.keras as keras`, please simply import keras with the statement `import keras` - the remainder of the code should stay the same

**Comment by levinwil** [comment]
@jdey4 @PSSF23 What do you both think about this? I don't know enough about the recruitment experiment to provide any meaningful feedback or to confidently approve.

**Review by PSSF23 — APPROVED** [review]
@levinwil I don't see anything wrong with the tutorial. How about we merge it first until changes are needed?

**Comment by levinwil** [comment]
@jdey4 @PSSF23 I have no idea if these results looks good or not, as I don't know anything about the recruitment experiment. Do you all have any thoughts?

**Comment by emilyachang** [comment]
@jdey4 @PSSF23 any updates on this? i want to make sure it's merged before the deadline.

**Comment by PSSF23** [comment]
@emilyachang please talk to @jdey4 and let him take a look. I and Will are not familiar with the recruitment experiment.

**Comment by emilyachang** [comment]
@PSSF23 according to jayanta, apparently jovo wants him to finish the paper before adding the recruitment tutorial and he also wants to run the code himself. so i guess we just leave this PR unmerged...?

**Review by PSSF23 — APPROVED** [review]
@emilyachang I believe leaving this PR unmerged would be too unappreciative of your work. So I would suggest merging this PR while not closing the related issues and not including it on the `rst` file until further confirmation.

@jdey4 @levinwil how about your opinions?

**Comment by levinwil** [comment]
I don't have a strong opinion, but I see no advantage in merging a PR that hasn't been approved by @jdey4. It could be great work or not - I just don't know enough about the experiment to provide any opinion. So I will defer to @jdey4
